src/OneBeyond.Studio.Crosscuts/Utilities/FileUploadValidators/FileValidatorBuilder.cs

```diff
@@ -80,6 +82,12 @@ public FileValidatorBuilder AllowVisio()
     public FileValidatorBuilder AllowSimpleText()
         => AddValidator(new TxtValidator());
 
+    public FileValidatorBuilder HasMaxSize(int maxFileSize)
```

Comment:
1. If this value is in `bytes`, it should probably be a `long`.
2. Would it be possible to have a utility function to perform the conversion for us? So

```csharp
.HasMaxSizeInMB(...)
.HasMaxSizeInGB(...)
```
or something else like that, to help developers use it?

Reply:
Done